### PR TITLE
Add ES ML Info API request

### PR DESCRIPTION
**Pull request by stefnestor**
@benwtrent @droberts195 per discussion, adds base ML info to ES Support Diag 🙏🏼

### CODE REVIEW

**Comment by stefnestor** [comment]
Ok, ready 🙏🏼

**Review by droberts195 — APPROVED** [review]
Thanks, LGTM, but please get approval from the maintainers too before merging

**Review by pickypg — APPROVED** [review]

**Pull request merged**
